library/ui-styles/src/main/res/values/styles_voice_message.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <attr name="vctr_voice_message_lock_background_style" format="reference" />
+    <attr name="vctr_voice_message_playback_background_style" format="reference" />
+    <attr name="vctr_voice_message_play_pause_button_background_style" format="reference" />
+    <attr name="vctr_voice_message_recording_playback_background_style" format="reference" />
+
+    <style name="VoiceMessageLockBackground">
+        <item name="android:layout_width">52dp</item>
+        <item name="android:layout_height">160dp</item>
+        <item name="android:background">@drawable/bg_voice_message_lock</item>
+        <item name="android:backgroundTint">?vctr_voice_message_lock_background</item>
+    </style>
+
+    <style name="VoiceMessageMicButton">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:background">?android:attr/selectableItemBackground</item>
+        <item name="android:layout_marginBottom">12dp</item>
+        <item name="android:layout_marginEnd">12dp</item>
+        <item name="android:src">@drawable/ic_voice_mic</item>
+    </style>
+
+    <style name="VoiceMessageSendButton">
+        <item name="android:layout_width">56dp</item>
+        <item name="android:layout_height">56dp</item>
+        <item name="android:background">@drawable/bg_send</item>
+        <item name="android:scaleType">center</item>
+        <item name="android:src">@drawable/ic_send</item>
+    </style>
+
+    <style name="VoiceMessageTimerIndicator">
+        <item name="android:layout_width">10dp</item>
+        <item name="android:layout_height">10dp</item>
+        <item name="android:layout_marginStart">20dp</item>
+        <item name="android:src">@drawable/circle</item>
+        <item name="tint">@color/palette_vermilion</item>
+    </style>
+
+    <style name="VoiceMessageTimer">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_marginStart">4dp</item>
+        <item name="android:lineSpacingExtra">4dp</item>
+        <item name="android:textSize">14sp</item>
+        <item name="android:textColor">@color/palette_gray_200</item>
+    </style>
+
+    <style name="VoiceMessageSlideToCancel">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:gravity">center</item>
+        <item name="android:textColor">@color/palette_gray_300</item>
+        <item name="android:textSize">14sp</item>
```

Comment:
You should not use textSize anymore. Please use style defined in `styles_text_view.xml`, or eventually inherit from one of those styles.

Reply:
Done